lib/src/values/logic_value.dart

```diff
@@ -699,7 +699,11 @@ abstract class LogicValue {
       return LogicValue.filled(other.width, LogicValue.x);
     }
 
-    if (this is _BigLogicValue || other is BigInt || other is _BigLogicValue) {
+    if (this is _BigLogicValue ||
```

Comment:
Why not just always check `this.width` and `other.width` and never check the type with `is`?

Also, let's use `_INT_BITS` instead of 64.  Eventually this may compile to javascript and then we'll only have 53 bits for ints.  Having fewer hard-coded 64's around will make things easier.